namedtensor/torch_base.py

```diff
@@ -99,6 +99,43 @@ def cat(tensors, dim):
                 tensors[i] = tensors[i]._force_order(tensors[0]._schema._names)
         return tensors[0]._new(torch.cat([t.values for t in tensors], dim=dim))
 
+    @staticmethod
+    def equal(tensor1, tensor2):
```

Comment:
I think we need to be more careful with the semantics here. The two tensors need to first have the same order before we check equality.

Reply:
Yeah, that makes much more sense. Do you think we should consider the 'masked' equality in this function, as the tensors which have the same names and 'masked' value but different names order would mask different dimensions?